tfx/examples/imdb/imdb_pipeline_native_keras.py

```diff
@@ -0,0 +1,196 @@
+# Lint as: python2, python3
```

Comment:
please do a side by side comparison for this file vs iris_pipeline_native_keras.py

other than tuner part, it should be about the same.

Reply:
fixed